### ISSUE TITLE
Channel View Issue

### ISSUE THREAD

**Issue by jshplayer** [apotdevin/thunderhub#399]
**Is your feature request related to a problem? Please describe.**
Whenever Tor browser is restarted it loses the channel view preferences

**Describe the solution you'd like**
Channel view preferences be made persistent via signed on user or stored within host not cookies

**Describe alternatives you've considered**
Not refreshing / closing browser

**Provide examples**
Closing browser will trigger this (tor browser - defaults)

**Additional context**

**Comment by apotdevin** [comment]
Yeah this doesn't work at the moment because Tor browser doesn't persist the local storage where these settings are saved. No simple solution for this. Will have to give it some thought

**Comment by jshplayer** [comment]
I figured as much, well is there any way for me to automate applied views or adjust some xml file to make it permanent / defaults? Adjusting them every session is a PITA especially with the new layout : )
